config.json

```diff
@@ -2659,6 +2659,14 @@
         "nugetId": "Xamarin.GooglePlayServices.Identity",
         "type": "xbd"
       },
+      {
+        "groupId": "com.google.android.gms",
+        "artifactId": "play-services-identity-credentials",
+        "version": "16.0.0-alpha04",
+        "nugetVersion": "116.0.0",
```

Comment:
This should be a pre-release `nugetVersion`.

Reply:
fixed